images/ceph/set-ceph-debug-level

```diff
@@ -105,14 +82,15 @@ exec_ceph_command() {
   
   # exec command
   for flag in "${CEPH_DEBUG_FLAG[@]}"; do
-    ARGS=("$action" global debug_"$flag")
-    if [[ "$debug_level" != "default" ]]; then
-      ARGS+=("$debug_level")
-    fi
-    # put stdout in /dev/null since increase debug log will overflow the terminal
-    echo "ceph config ${ARGS[*]}"
-    ceph config "${ARGS[@]}" &> /dev/null & pids+=($!)
-    
+    for daemon in "${DAEMONS[@]}"; do
```

Comment:
Is the nested `for` loop intended? Looks like we'll end up with over 100 commands (4 * 27). Seems like we don't want the full combination, or else I'm misunderstanding.

Reply:
This is the intention.